### DISCUSSION TITLE
Error: Invalid hook call.

### DISCUSSION THREAD

**Discussion by sivalive123** [JedWatson/react-select#4817] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/38655636/134921178-8b9f1928-29e5-4e56-b6ce-023961f24ed7.png)

![image](https://user-images.githubusercontent.com/38655636/134917811-c3272fc7-bf45-4efc-95c8-a79f7f95dc46.png)
Error: Invalid hook call. Hooks can only be called inside of the body of a function component.

I am getting this error. Its happening only react-select. For other libraries i am not getting this error.

Can some pls help ?

**Comment by ebonow** [comment, marked as answer]
@sivalive123 have you looked in your lock file to see if your app is using multiple versions of React or ReactDOM?

**Reply by sivalive123** [reply]
Yes. I resolved by removing multiple versions of React. Thanks !!!
